app/helpers/navigation_items.rb

```diff
@@ -31,7 +31,12 @@ def for_support_primary_nav(current_support_user, current_controller)
     end
 
     def for_support_account_nav(current_support_user)
-      if current_support_user
+      if (impersonated_user = current_support_user&.impersonated_provider_user)
```

Comment:
This breaks when not logged in because `current_support_user` returns `false`, not `nil`

Reply:
I wonder if it should return `nil` anyway?

Reply:
Hmm yes. Thought about changing `current_support_user` to return `nil` (feels more natural) but this will break memoization. Fixed `navigation_items.rb` instead.